ppb/scenes.py

```diff
@@ -99,15 +96,16 @@ def remove(self, game_object: Hashable) -> None:
             s.discard(game_object)
 
 
-class BaseScene(Scene, EventMixin):
+class BaseScene(EventMixin):
     # Background color, in RGB, each channel is 0-255
     background_color: Sequence[int] = (0, 0, 100)
     container_class: Type = GameObjectCollection
 
-    def __init__(self, engine, *,                 
-                 set_up: Callable=None, pixel_ratio: Number=64,
+    def __init__(self, engine, *,
+                 set_up: Callable = None, pixel_ratio: Number = 64,
                  **kwargs):
-        super().__init__(engine)
+        super().__init__()
```

Comment:
Remove the call to `super().__init__`, `EventMixin` has no `__init__` method.

Reply:
Technically not true, because of C3 linearization.

However, I would be pretty ok with removing the `engine` mandatory positional argument and therefore making the `__init__` signature up and down the tree more predictable.

Reply:
I think the engine mandatory argument is a thing that's been on my list to kill for a couple of versions now, do so if you'd like.

Reply:
Ok, see https://github.com/astronouth7303/pursuedpybear/pull/2